samples/dfu/README.rst

```diff
@@ -7,44 +7,70 @@ Overview
 This DFU application demonstrates how to connect with Golioth and use Device
 Firmware Upgrade (DFU) procedure.
 
+The app is first built and flashed to the device which will then listen to
+Golioth Cloud for a new firmware binary to become available. A second binary is
+built with a new version number, registered as an artifact, and depolyed as a
+release. The device will detect this change and automatically download and
+install.
+
 Requirements
 ************
 
 - Golioth credentials
 - Network connectivity
 
-Using nRF9160 Feather with nRF Connect SDK
-******************************************
+Using nRF9160 Feather or Thingy:91 with nRF Connect SDK
+*******************************************************
+
+A full nRF9160 firmware `flashing overview`_ is available in the Golioth docs.
+Please use that resource to install the tools necessary to flash these devices.
```

Comment:
What do you want to reference here? IMO Golioth docs should refer to SDK, not the other way.

Reply:
> I think that information is too much to be included in the readme, 

I agree.

> but since we have documentation on it, we should make that known.

There are plenty of sites that already document that. I would rather stick closer to the source/vendor, i.e. reference Nordic (author of nrfjprog) and Jared (modified newtmgr with downstream patches) documentation instead.

Do we really want to break "Golioth docs ------> Golioth Zephyr SDK ------> Zephyr RTOS" one way relation? Isn't that going to make us pain later when synchronizing one to the other?